player/stream-test/index.html

```diff
@@ -1,11 +1,18 @@
 <script type="text/javascript" src="https://cdn.bitmovin.com/player/web/8/bitmovinplayer.js"></script>
 <script type="text/javascript" src="https://cdn.bitmovin.com/analytics/web/2/bitmovinanalytics.min.js"></script>
+<script src="https://cdn.jsdelivr.net/npm/bitmovin-player-ui@4.0.0-a.1/dist/js/bitmovinplayer-ui.js"></script>
+<link href="https://cdn.jsdelivr.net/npm/bitmovin-player-ui@4.0.0-a.1/dist/css/bitmovinplayer-ui.css" rel="stylesheet">
 
 <hr class="mt-5">
 
 <div class="demo-row">
-    <div class="demo-video tv-frame">
-        <div id="player">
+    <div class="demo-video-wrapper">
+        <div class="demo-video tv-frame">
+            <div id="player">
+            </div>
+        </div>
+        <div class="demo-video-footnote">
+            * This demo is using a pre-release version of the Bitmovin Player UI. <a href="https://community.bitmovin.com/t/new-bitmovin-player-ui-is-comming/3800" target="_blank">Learn more</a>
```

Comment:
Seems like the URL has changed and is redirecting to `https://community.bitmovin.com/t/new-default-player-ui/3800`, should we just link to that one instead?

Reply:
[8da08a0](https://github.com/bitmovin/demos/pull/131/commits/8da08a0fd4d6a5fd5989b67544e9c83664ece6b4)